include/dfu/dfu_target_suit.h

```diff
@@ -29,23 +29,22 @@ extern "C" {
 int dfu_target_suit_set_buf(uint8_t *buf, size_t len);
 
 /**
- * @brief See if data in buf indicates SUIT style upgrade.
+ * @brief Initialize dfu target for specific image, perform steps necessary to receive firmware.
  *
- * Not implemented as it does not currently have any use cases.
- * Currently always returns -ENOSYS.
- *
- * @retval -ENOSYS
- */
-bool dfu_target_suit_identify(const void *const buf);
-
-/**
- * @brief Initialize dfu target, perform steps necessary to receive firmware.
+ * If you call this function, you must call dfu_target_suit_done() to finalize the firmware upgrade
+ * before initializing any other images.
  *
  * @param[in] file_size Size of the current file being downloaded.
- * @param[in] img_num Image pair index.
+ * @param[in] img_num Image index.
  * @param[in] cb Callback for signaling events(unused).
  *
- * @retval 0 If successful, negative errno otherwise.
+ * @return 0 If successful
+ * @return -ENODEV errno code if the buffer has not been initialized.
+ * @return -ENXIO errno code if the partition dedicated for provided image is not found.
+ * @return -ENOMEM errno code if the buffer is not large enough.
+ * @return -EFAULT errno code if flash device assigned to the image is not available on the device.
+ * @return -EBUSY errno code if the any image is already initialized and stream flash is in use.
+ * @return other negative errno code if the initialization failed.
```

Comment:
Doxygen will parse the first word after the value (`errno`) as the first word of the description, so maybe:
```suggestion
 * @return -ENODEV_errno_code If the buffer has not been initialized.
 * @return -ENXIO_errno_code If the partition dedicated for provided image is not found.
 * @return -ENOMEM_errno_code If the buffer is not large enough.
 * @return -EFAULT_errno_code If flash device assigned to the image is not available on the device.
 * @return -EBUSY_errno_code If the any image is already initialized and stream flash is in use.
 * @return other_negative_errno_code If the initialization failed.
```

Reply:
hmm, I guess these should be without `_` because they are specific types of errno code, not specific macros. Or maybe just `errno`